howitworks.rst

```diff
@@ -74,11 +74,11 @@ libreport
 """""""""
 
 Libraries providing an API for reporting problems
-via different paths like email, bugzilla, faf, scp upload..
+via different paths like email, bugzilla, ABRT Analytics, scp upload..
```

Comment:
Not directly related but maybe capitalize Bugzilla (and possibly other proper names)? The capitalized "ABRT Analytics" looks kind of odd when all the rest is all lowercase.

Reply:
Nod. But I would like to do it in different commit/PR